src/sdk/accountspace_nb.js

```diff
@@ -87,19 +84,19 @@ class AccountSpaceNB {
     async get_user(params, account_sdk) {
         const action = IAM_ACTIONS.GET_USER;
         const requesting_account = system_store.get_account_by_email(account_sdk.requesting_account.email);
-        const account_name = account_util.get_account_name_from_username(params.username, requesting_account.name.unwrap());
+        const account_name = account_util.get_account_name_from_username(params.username, requesting_account._id);
         const requested_account = system_store.get_account_by_email(account_name);
         account_util._check_if_requesting_account_is_root_account(action, requesting_account,
                 { username: params.username, iam_path: params.iam_path });
         account_util._check_if_account_exists(action, account_name);
         account_util._check_if_requested_account_is_root_account_or_IAM_user(action, requesting_account, requested_account);
         account_util._check_if_requested_is_owned_by_root_account(action, requesting_account, requested_account);
+        const iam_arn = iam_utils.create_arn_for_user(requesting_account._id.toString(), params.username, requested_account.iam_path);
```

Comment:
For the function `create_arn_for_user` you must pass the `iam_path`, suggesting to pass the default, something like: 

```suggestion
        const iam_arn = iam_utils.create_arn_for_user(requesting_account._id.toString(), params.username, requested_account.iam_path || IAM_DEFAULT_PATH);
```